packages/web/src/components/nav/desktop/PlaylistLibrary.tsx

```diff
@@ -239,6 +241,110 @@ const PlaylistFolderNavItem = ({
   )
 }
 
+type PlaylistNavItemProps = {
+  playlist: AccountCollection
+  url: string
+  addTrack: (trackId: ID) => void
+  isOwner: boolean
+  onReorder: (
+    draggingId: ID | SmartCollectionVariant,
+    droppingId: ID | SmartCollectionVariant
+  ) => void
+  hasUpdate?: boolean
+  dragging: boolean
+  draggingKind: string
+  onClickPlaylist: (id: ID, hasUpdate: boolean) => void
+  onClickEdit?: (id: ID) => void
+}
+const PlaylistNavItem = ({
+  playlist,
+  hasUpdate = false,
+  url,
+  addTrack,
+  isOwner,
+  onReorder,
+  dragging,
+  draggingKind,
+  onClickPlaylist,
+  onClickEdit
+}: PlaylistNavItemProps) => {
+  const { id, name } = playlist
+  const [isHovering, setIsHovering] = useState(false)
+
+  return (
+    <Droppable
+      key={id}
+      className={navColumnStyles.droppable}
+      hoverClassName={navColumnStyles.droppableHover}
+      onDrop={addTrack}
+      acceptedKinds={['track']}
+      disabled={!isOwner}
+    >
+      <PlaylistNavLink
+        droppableKey={id}
+        playlistId={id}
+        name={name}
+        link={url}
+        to={url}
+        onReorder={onReorder}
+        isActive={() => url === getPathname()}
+        activeClassName='active'
+        className={cn(navColumnStyles.link, {
+          [navColumnStyles.playlistUpdate]: hasUpdate,
+          [navColumnStyles.droppableLink]:
+            isOwner &&
+            dragging &&
+            (draggingKind === 'track' || draggingKind === 'playlist'),
+          [navColumnStyles.disabledLink]:
+            dragging &&
+            ((draggingKind !== 'track' &&
+              draggingKind !== 'playlist' &&
+              draggingKind !== 'library-playlist') ||
```

Comment:
I noticed `droppableLink` isn't checking for 'library-playlist' - intentional?

Reply:
yeah I noticed that too, maybe @raymondjacobson knows? I just copied this over - i assume it is intentional though

Reply:
This might help:
* You can drag a track onto a playlist in your nav (the playlist name is purple)
* You can drag a playlist onto your left nav entirely (the playlists in the left nav are purple)
* You can drag a library-playlist (one that's already in your nav bar) around, but it lets you reorder rather than lets you drop it into your nav

<img width="570" alt="Screen Shot 2022-03-01 at 6 14 36 PM" src="https://user-images.githubusercontent.com/2731362/156281799-5e86c50b-7602-450c-949e-0cb1b123174c.png">
<img width="640" alt="Screen Shot 2022-03-01 at 6 14 22 PM" src="https://user-images.githubusercontent.com/2731362/156281795-8638496a-27e2-4495-8d31-117fd88adef5.png">
<img width="652" alt="Screen Shot 2022-03-01 at 6 14 08 PM" src="https://user-images.githubusercontent.com/2731362/156281789-bc50d22c-f88a-440d-8369-0644fbe81eaf.png">

Reply:
haha this makes sense, thanks for clarifying